### DISCUSSION TITLE
How to add a robot urdf for object nav task?

### DISCUSSION THREAD

**Discussion by sai-prasanna** [facebookresearch/habitat-lab#1040] [Q&A]
Hi, I am trying to build a new custom task extending the object nav task. I want to make the agent look like a fetchbot instead of a invisible cylinder. I would appreciate it if anyone helps me on doing this. Ideally I want to replace the invisible cylinder with fetchbot and use the fetchbot's sensors and actuators as the main agent for navigation.

**Comment by aclegg3** [comment]
Hey @sai-prasanna 
Get anything figured out here yet?

I think there is a short answer (pointer to habitat-sim API to get you moving) and a long answer (how to modify lab configs and such to do this in a more generic way).
